ldk/go/filesystemService.go

```diff
@@ -2,25 +2,54 @@ package ldk
 
 import (
 	"context"
+	"os"
 	"time"
 
 	"github.com/open-olive/loop-development-kit/ldk/go/proto"
 )
 
 // FilesystemService is an interface that defines what methods plugins can expect from the host
 type FilesystemService interface {
-	Dir(context.Context, string) ([]FileInfo, error)
+	Dir(context.Context, string) ([]os.FileInfo, error)
 	ListenDir(context.Context, string, ListenDirHandler) error
-	File(context.Context, string) (FileInfo, error)
```

Comment:
I noticed that the "one time query for FileInfo" function is missing now. Is that basically just moved to the `Stat` on the `File` interface?

Reply:
Correct.  You can get the FileInfo by opening a file and then calling `.Stat()`

Reply:
👍